aspnetcore/signalr/authn-and-authz/sample/Startup.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.IdentityModel.Tokens;
+using SignalRAuthenticationSample.Data;
+using SignalRAuthenticationSample.Hubs;
+
+namespace SignalRAuthenticationSample
+{
+    public class Startup
+    {
+        // We use a key generated on this server during startup to secure our tokens.
+        // This means that if the app restarts, existing tokens become invalid. It also won't work
+        // when using multiple servers.
+        public static readonly SymmetricSecurityKey SecurityKey = new SymmetricSecurityKey(Guid.NewGuid().ToByteArray());
+
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddDbContext<ApplicationDbContext>(options =>
+                options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection")));
+
+            services.AddIdentity<ApplicationUser, IdentityRole>()
+                .AddEntityFrameworkStores<ApplicationDbContext>()
+                .AddDefaultTokenProviders();
+
+            services.AddAuthentication(options =>
+                {
+                    // Identity made Cookie authentication the default, we want JWT Bearer Auth to be the default.
```

Comment:
Code looks close, spit up comments
![image](https://user-images.githubusercontent.com/3605364/42249126-bf7a60e8-7ee6-11e8-9ba7-ebc397c26ba1.png)

Reply:
The official guidance is to not exceed 85 characters per line.

Reply:
"spit up comments" :joy: :joy: :joy: :joy:

Reply:
@rachelappel The hack-a-loogie variety. 😄